packages/babel-traverse/src/path/modification.js

```diff
@@ -24,20 +24,18 @@ export function insertBefore(nodes) {
     (this.parentPath.isForStatement() && this.key === "init")
   ) {
     if (this.node) nodes.push(this.node);
-    this.replaceExpressionWithStatements(nodes);
+    return this.replaceExpressionWithStatements(nodes);
   } else if (Array.isArray(this.container)) {
     return this._containerInsertBefore(nodes);
   } else if (this.isStatementOrBlock()) {
     if (this.node) nodes.push(this.node);
-    this._replaceWith(t.blockStatement(nodes));
+    return this._replaceWith(t.blockStatement(nodes));
```

Comment:
I think in this case were we implicitly create a `BlockStatement` we should not return the block statement but instead only the inserted nodes. I was thinking about what the usecase could be  that someone might want to know if this was created, but I think it does not have any value to return the BlockStatement itself.

Assuming this two examples

```js
if (true) console.log('');

if (true) {
  console.log('');
}
```

And you always want to insert a Statement before or after `console.log()` I think it should be consistent and always return the path to the nodes that were inserted.
Right now `inspertBefore/After()` would in this two cases still return different results.